pkg/target/regolib/src.rego

```diff
@@ -245,11 +282,19 @@ matches_nsselector(match) {
 }
 
 matches_nsselector(match) {
+  not is_ns(input.review.kind)
   has_field(match, "namespaceSelector")
   get_ns[ns]
   matches_namespace_selector(match, ns)
 }
 
+# if we are matching against a namespace, match against either the old or new object
+matches_nsselector(match) {
+  is_ns(input.review.kind)
+  has_field(match, "namespaceSelector")
+  any_labelselector_match(get_default(match, "namespaceSelector", {}), ns)
```

Comment:
maybe I missed this somewhere, but why does `any_labelselector_match` have 2 args here?

Reply:
Ya, def shouldn't be there. I'm guessing it was compiling because it was assigning the output of `any_labelselector_match` to `ns` because rego allows functions to have the output either provided via the equals sign or to the last argument.

Removed.